.github/workflows/publish-harness.yaml

```diff
@@ -173,3 +173,76 @@ jobs:
             HARNESS_BASE_IMAGE_REPO=ghcr.io/pluralsh/stackrun-harness-base
             HARNESS_BASE_IMAGE_TAG=${{ needs.publish-harness-base.outputs.version }}
             
+  publish-harness-ansible:
+    name: Build and push harness ansible container
+    runs-on: ubuntu-20.04
+    needs: [publish-harness-base]
+    strategy:
+      matrix:
+        versions:
+          - ansible: '4.10.0'
```

Comment:
For terraform I primarily used major and minor (w/o patch) versions to simplify the user facing config, we should keep that pattern for ansible too. You'd just add another key like `tag` and plumb that in to the docker-metadata action instead of the full ansible version